karma-story-editor.config.cjs

```diff
@@ -152,8 +152,8 @@ module.exports = function (config) {
         defaultViewport: getViewport(config.viewport),
       },
     },
-
     client: {
+      clientDisplayNone: config.snapshots || false,
```

Comment:
I suppose with the banner removed in `packages/karma-fixture/src/client_with_context.html` we don't need this anymore? Then we can revert all changes to this file.